src/server.ts

```diff
@@ -2,37 +2,142 @@ import express, { Request, Response, Application, NextFunction } from 'express';
 import { config } from './config';
 import logger from './logger';
 import { v4 as uuid } from 'uuid';
+import axios from 'axios';
+import * as _ from 'underscore';
 
+const url = 'http://www.omdbapi.com?apikey=16d8c738&t=';
 const app: Application = express();
 const PORT = config.APP_PORT;
 const ENV = config.ENV;
+let movies: any[] = [];
+
+const requestLogger = (req: Request, res: Response, next: NextFunction) => {
+  const {
+    route: { path },
+    method,
+    body,
+    query,
+    headers,
+  } = req;
+
+  logger.info(`${method} ${path} ${headers['x-request-id'] || uuid()}`);
 
-app.use((req: Request, res: Response, next: NextFunction) => {
-  const requestId = req.headers['x-request-id'] || uuid();
-  logger.info(`${req.method}  ${req.path}  ${req.query.foo}   ${requestId}`);
-  res.set('X-Request-ID', requestId);
   next();
-});
+};
+
+const errorHandler = (
+  err: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  logger.error(err);
+  res.status(500).send(err.message);
+
+  next();
+};
+
+const verifyMoviePresent = (id: string) => {
```

Comment:
Can't see any usage of this function, do we need actually need it?

Reply:
No. Removed